hal/armv7m/imxrt/10xx/console.c

```diff
@@ -15,6 +15,22 @@
 
 #include <hal/hal.h>
 
+#define CONCAT(a, b)         a##b
+#define CONCAT2(a, b)        CONCAT(a, b)
+#define UART_PIN(n, pin)     CONCAT(UART, n##_##pin)
+#define CONSOLE_BASE(n)      UART_PIN(n, BASE)
+#define CONSOLE_CLK(n)       UART_PIN(n, CLK)
+#define CONSOLE_MUX(n, pin)  CONCAT2(pctl_mux_gpio_, UART_PIN(n, pin))
+#define CONSOLE_PAD(n, pin)  CONCAT2(pctl_pad_gpio_, UART_PIN(n, pin))
+#define CONSOLE_ISEL(n, pin) CONCAT2(pctl_isel_lpuart, CONCAT(n, _##pin))
+#define CONSOLE_BAUD(n)      (UART_PIN(n, BAUDRATE))
+
+#if CONSOLE_BAUD(UART_CONSOLE) == 0
+#undef CONSOLE_BAUD
```

Comment:
I like previous version better. `undef` is one bridge too far

Reply:
Ok.